README.md

```diff
@@ -27,12 +27,27 @@ You should see output similar to the following:
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
```

Comment:
Your README still mentions setup.sh and you still have it around but you shouldn't need it anymore.
All you need from it is `cp -n .env.template .env` the first time someone clones the repo.